src/Controls/Foldable/src/Controls.Foldable.csproj

```diff
@@ -22,6 +22,13 @@
     <Description>Foldable or multi-screen device support for .NET Multi-platform App UI (.NET MAUI) apps.</Description>
   </PropertyGroup>
   <Import Project="$(MauiSrcDirectory)MultiTargeting.targets" />
+  <ItemGroup>
+    <!-- Security SBOM targets -->
+    <PackageReference Include="Microsoft.Sbom.Targets" Version="$(MicrosoftSbomTargetsPackageVersion)">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>    
```

Comment:
Maybe this can be moved to the Directory.Build.targets (or some other better file @rmarinho?) and then just include  if the `IsPackable` property is true?

@rmarinho probably can quickly say what the best option here is now with all the arcade fanciness.

Reply:
yes. This is something I wasn't sure where to place. Sorry

Reply:
> @rmarinho probably can quickly say what the best option here is now with all the arcade fanciness.

Sure waiting for input.

Reply:
yap I agree we should maybe add to the NugetVersion targets ? 

https://github.com/dotnet/maui/blob/main/eng/NuGetVersions.targets#L312